lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/DBOMembershipInvtnSubmissionDAOImpl.java

```diff
@@ -143,6 +142,16 @@ public MembershipInvtnSubmission get(String id) throws DatastoreException, NotFo
 		return dto;
 	}
 
+	@WriteTransactionReadCommitted
```

Comment:
I don't think this has any effect.  the transaction is established at the manager level.